src/test/kotlin/no/nav/familie/ef/iverksett/oppgave/OpprettOppgaveForBarnServiceTest.kt

```diff
@@ -0,0 +1,145 @@
+package no.nav.familie.ef.iverksett.oppgave
+
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkObject
+import io.mockk.verify
+import no.nav.familie.ef.iverksett.iverksetting.IverksettingRepository
+import no.nav.familie.ef.iverksett.iverksetting.domene.Barn
+import no.nav.familie.ef.iverksett.iverksetting.domene.Iverksett
+import no.nav.familie.util.FnrGenerator
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+
+/**
+ * Vi tar utgangspunkt i fra dato (men ikke fra og med), og til og med dato, ved sjekk om en fødselsdato slår til i løpet av
+ * k antall uker.
+ */
+internal class OpprettOppgaveForBarnServiceTest {
+
+    val iverksett = mockk<Iverksett>()
+    val iverksettingRepository = mockk<IverksettingRepository>()
+    val oppgaveClient = mockk<OppgaveClient>()
+    val opprettOppgaveForBarnService = OpprettOppgaveForBarnService(oppgaveClient, iverksettingRepository)
+
+    @BeforeEach
+    fun init() {
+        mockkObject(OppgaveUtil)
+        mockkObject(OppgaveBeskrivelse)
+        every { oppgaveClient.opprettOppgave(any()) } returns 0L
+        every { OppgaveUtil.opprettOppgaveRequest(any(), any(), any()) } returns mockk()
+        every { OppgaveBeskrivelse.beskrivelseBarnBlirSeksMnd() } returns ""
+        every { OppgaveBeskrivelse.beskrivelseBarnFyllerEttÅr() } returns ""
+    }
+
+    @Test
+    fun `barn blir seks mnd om 4 dager, sjekk om fyller innen 1 uke, forvent kall til beskrivelseBarnBlirSeksMnd`() {
+        val fødselsdato = LocalDate.now().minusMonths(6).plusDays(3)
+        every { iverksettingRepository.hentAlleBehandlinger() } returns listOf(iverksett)
+        every { iverksett.søker.barn } returns listOf(opprettBarn(generateFnr(fødselsdato)))
+        opprettOppgaveForBarnService.opprettOppgaverForAlleBarnSomFyller(innenAntallUker = 1)
+        verify { OppgaveBeskrivelse.beskrivelseBarnBlirSeksMnd() }
```

Comment:
Litt "risky" sjekk - hvis det kommer en ny oppgavebeskrivelsestype så gjelder ikke denne lenger. 
Bør vi ikke kjøre verify på `oppgaveClient.opprettOppgave` og evt bruke `slot` for å verifisere innholdet?

Reply:
Hmm, men hva hvis innholdet endres ?

Reply:
Da feiler testen? Det er jo bra? :)

Reply:
Nå blir dette litt annerledes siden beskrivelsene ligger i ef-sak, og siden oppgavene opprettes i iverksett. Setter derfor denne til resolved her.